src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -3776,6 +3776,7 @@ .maxstack  1
         /// </summary>
         [WorkItem(825903, "DevDiv")]
         [Fact]
+        [Trait("Require32", "true")]
```

Comment:
Yes.  I'm trying to figure out a couple of items here:

- How do we at least semi-regularly run this version so that we don't regress.
- How to we burn down on the failing items to get to a purely clean run.

The result will be bugs filed for these tests in one way or the other.